### ISSUE TITLE
Add UI element to sync/open project in Data Tracker 

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#7777]
Per [discussion](https://github.com/cityofaustin/atd-data-tech/issues/7566#issuecomment-965855419), a ~button~ link that will serve to push data to Knack and then function as a link to the signals for a given project in Data Tracker. 

Before sync:
![Summary - Sync to Data Tracker](https://user-images.githubusercontent.com/1463708/146102139-22bbe528-a214-460c-a926-f3159caf9b7b.png)

After:
![Summary Page – 6 0](https://user-images.githubusercontent.com/1463708/146102260-e3abc657-6aa5-4f6d-a26f-0f4d8b51aa45.png)

**Comment by johnclary** [comment]
We can use this pattern for construction the URL to the Data Tracker project details:

`https://atd.knack.com/amd#projects/project-details/<knack-id>`

this page 👆  is going to need to be [replaced with a simplified project details page](https://github.com/cityofaustin/atd-data-tech/issues/7776), but we should be able to gracefully swap out a different knack page and use the same URL as above. Right, @zappfyllc?

**Comment by amenity** [comment]
@johnclary @frankhereford - y'all mentioned the user "syncing" when they updated the project... We will want these kept synchronized: 
- Signals included in the project 
- Project name

I think all other project data can just live in Moped, right?

**Comment by zappfyllc** [comment]
@amenity this initial scope is seen as a one-time synchronization in order to store the primary keys for the records on each database (Moped and Knack) so that full synchronization can occur via ETL in #7568. @frankhereford and @johnclary can both correct me if I'm wrong, but I believe @frankhereford has written the implementation in a way where much of the code he uses via a client-side call from the Moped editor will be able to be reused in other synchronization patterns between Moped and Knack.

Unfortunately the just-in-time updating of the project's attributes between Moped and Knack would require a more complex implementation (although was discussed in the parent epic with both Sergio and John), so this is sort of a piecemeal approach to getting there since the primary keys (the `project_id` in Moped and the knack record identifier in Knack) will not change and allow us to synchronize between the two datasets without complicated or unreliable lookup values.

**Comment by amenity** [comment]
@zappfyllc @frankhereford @patrickm02L @johnclary Throwing this out bc it's what I imagined during our discussion, but totally open to other approaches. 


![Summary Page – 5 0 – Sync to Data Tracker](https://user-images.githubusercontent.com/1463708/144162058-cc1da168-c077-4ab7-989b-7def645ce5cc.png)


![Summary Page – 5 0](https://user-images.githubusercontent.com/1463708/144162109-99831c9d-1f95-48a9-a310-0a615d7f553e.png)

**Comment by amenity** [comment]
Just spoke with @johnclary and confirmed that there will actually be a page in Data Tracker for this, see #7776

**Comment by zappfyllc** [comment]
@amenity I actually imagined the same thing as what you have here, but understand the request @johnclary is making in #7776 means we would want a single project link instead of capturing the signal ID links. My only concern with that direction is having two projects tables, one in Moped and one in Data Tracker, for the worry that if users have two places to go, they will default to Data Tracker and not use Moped. That might be an illegitimate concern but I want to express it and emphasize that your design above nicely captures the ability to switch between Data Tracker and Moped without adding another projects table/page. This is ideal to me because it directs AMD users to go to Data Tracker for very specific signal asset details while maintaining high level project tracking in Moped.

**Comment by amenity** [comment]
@zappfyllc I think I agree... @johnclary - do we really need to have this page in Data Tracker?

**Comment by zappfyllc** [comment]
@johnclary FWIW when @amenity and I discussed the latest Signal Projects Dashboard with Renee and Ivonne (I believe this was after you had left the call for another meeting), they seemed to understand they would lose their current Project Details page because they mostly focused on the Signal Details page and where we could draw in project associations/relationships from within that page rather than the Project Details page.

We also discussed the possibility of sending over some information from Moped to Data Tracker when a project is completed. This would allow them to visit the Signal Details page and see some information from projects involving that specific signal or at least a link between that project in Moped (which would land them on the Moped Project Details). It might be a nice and consistent pattern to have everything in the context of a Signal in Data Tracker, while maintaining all context for a Project in Moped, but I understand we may then need to identify any information we'd currently lose by removing the Project Details page in Data Tracker and if we might solve that in the Knack integration.
